docs/get-started/with-python.md

```diff
@@ -236,18 +236,33 @@ print(pred_onx)
 
 ## Python API Reference Docs
 
- <span class="fs-5"> [Go to the ORT Python API Docs](../api/python/api_summary.html){: .btn  .mr-4 target="_blank"} </span> 
+ <span class="fs-5"> [Go to the ORT Python API Docs](../api/python/api_summary.html){: .btn  .mr-4 target="_blank"} </span>
 
 ## Builds
-If using pip, run pip install `--upgrade pip` prior to downloading.	 
+If using pip, run `pip install --upgrade pip` prior to downloading.
 
 | Artifact      | Description | Supported Platforms |
 |-----------    |-------------|---------------------|
 |[onnxruntime](https://pypi.org/project/onnxruntime)|CPU (Release)| Windows (x64), Linux (x64, ARM64), Mac (X64),  |
-|[ort-nightly](https://test.pypi.org/project/ort-nightly)|CPU (Dev)    | Same as above |
+|[ort-nightly](https://aiinfra.visualstudio.com/PublicPackages/_artifacts/feed/ORT-Nightly/PyPI/ort-nightly)|CPU (Dev)    | Same as above |
 |[onnxruntime-gpu](https://pypi.org/project/onnxruntime-gpu)|GPU (Release)| Windows (x64), Linux (x64, ARM64) |
-|[ort-nightly-gpu](https://test.pypi.org/project/ort-nightly-gpu)|GPU (Dev) | Same as above |
+|[ort-nightly-gpu for CUDA 11.*](https://aiinfra.visualstudio.com/PublicPackages/_artifacts/feed/ORT-Nightly/PyPI/ort-nightly-gpu) |GPU (Dev) | Windows (x64), Linux (x64, ARM64) |
+|[ort-nightly-gpu for CUDA 12.*](https://aiinfra.visualstudio.com/PublicPackages/_artifacts/feed/ort-cuda-12-nightly/PyPI/ort-nightly-gpu) |GPU (Dev) | Windows (x64), Linux (x64, ARM64) |
```

Comment:
We might need to hold on it a moment. Jian has to create a new feed for the CUDA 12 packages, because the one referenced here already contains a 1.17.0 package but our 1.17.0 release is not prepared yet. We cannot just delete the package and publish a new one there because each package is immutable. We need to create a new feed instead.

Reply:
I think nightly version shall be 1.17.0.